09.mesh-modeler/02.item-editor/docs.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Item Editor'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+[TOC]
+
+The Item Editor allows you to edit items (.Item.Gbx files) that can be placed in maps. It can be accessed directly from the *Title tools* menu, but it is recommended to open it from the map editor, via the *Item Edition* panel, at the left bottom corner of the screen in Item mode.
+
+Let's go through the list of parameters you can set.
+
+
+## Pivots
+
+When you place an item in the map, the mouse cursor is aiming at a particular point in the map. But how to place the item around this point? If your item is a cube, should the editor place the item so that the cursor point is at the center of the cube ? Or the center of the bottom face? Or one corner of the cube?
```

Comment:
```suggestion
When you place an item in the map, the mouse cursor is aiming at a particular point in the map. But how should the item be placed compared to this point? If your item is a cube, should the editor place the item so that the cursor point is at the center of the cube ? Or the center of the bottom face? Or one corner of the cube?
```